ansible/configs/ocp4-disconnected-osp-lab/env_vars.yml

```diff
@@ -0,0 +1,578 @@
+      description: "mDNS"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 80
+      port_range_max: 80
+      description: "Ingress HTTP"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 443
+      port_range_max: 443
+      description: "Ingress HTTPS"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 1936
+      port_range_max: 1936
+      remote_ip_prefix: "{{ ocp_network_subnet_cidr }}"
+      description: "router stats"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 4789
+      port_range_max: 4789
+      remote_group: "master_sg"
+      description: "VXLAN from master"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 4789
+      port_range_max: 4789
+      remote_group: "worker_sg"
+      description: "VXLAN"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 6081
+      port_range_max: 6081
+      remote_group: "master_sg"
+      description: "Geneve from master"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 6081
+      port_range_max: 6081
+      remote_group: "worker_sg"
+      description: "Geneve"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "worker_sg"
+      description: "Worker ingress internal (tcp)"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "master_sg"
+      description: "Worker ingress from master (tcp)"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "master_sg"
+      description: "Worker ingress from master (udp)"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 9000
+      port_range_max: 9999
+      remote_group: "worker_sg"
+      description: "Worker ingress internal (udp)"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 10250
+      port_range_max: 10250
+      remote_group: "master_sg"
+      description: "master ingress kubelet secure from master"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 10250
+      port_range_max: 10250
+      remote_group: "worker_sg"
+      description: "master ingress kubelet secure"
+    - protocol: tcp
+      direction: ingress
+      port_range_min: 30000
+      port_range_max: 32767
+      remote_group: "worker_sg"
+      description: "worker ingress services (tcp)"
+    - protocol: udp
+      direction: ingress
+      port_range_min: 30000
+      port_range_max: 32767
+      remote_group: "worker_sg"
+      description: "worker ingress services (udp)"
+    - protocol: vrrp
+      direction: ingress
+      remote_ip_prefix: "{{ ocp_network_subnet_cidr }}"
+      description: "VRRP"
+
+
+# These will influence the bastion if it is being deployed
+bastion_instance_type: 2c2g30d
+bastion_instance_image: rhel-guest-7.7u2
```

Comment:
The image is not available clusterwide ?

```
Resource UPDATE failed: resources.utilityvm: Resource CREATE failed: resources[0]: Property error: resources.instance.properties.block_device_mapping_v2[0].image: Error validating value 'rhel-guest-7.7u2': No images matching {'name': u'rhel-guest-7.7u2'}.
```

Reply:
Can we do something like:
- check if the image (or instance flavor) exists
- if it doesn't exist: create it
- if it exists: continue

?

Reply:
This should be run against the new blue cluster, where thee images and flavors exist.

To address this specific comment, as discussed, we will add a role that takes care of ensuring the flavor(s) exists. For the image, it should be a standard part of the OpenStack cluster build that the infra team is responsible for - but we can look at including that down the road in the config as a stretch goal.